R/run_PTXQC.R

```diff
@@ -0,0 +1,51 @@
+##
```

Comment:
please do not add this file to version control, i.e. `git rm --cached R/run_PTXQC.R`